src/CMakeLists.txt

```diff
@@ -304,6 +305,9 @@ if(BUILD_DEV)
     target_compile_definitions(migraphx PUBLIC -DBUILD_DEV)
 endif()
 
+file(STRINGS "${CMAKE_SOURCE_DIR}/test/onnx/.onnxrt-commit" String_output)
+target_compile_definitions(migraphx PUBLIC MIGRAPHX_ORT_SHA1="${String_output}")
```

Comment:
This should be added to the driver not to migraphx.

Reply:
Done and Done